.eslintrc

```diff
@@ -1,9 +1,76 @@
 {
-  "parser": "typescript-eslint-parser",
-  "extends": ["plugin:jsx-a11y/recommended"],
-  "plugins": ["jsx-a11y", "tree-shaking"],
+  "parser": "@typescript-eslint/parser",
+  "parserOptions": {
+    "project": "./tsconfig.json"
+  },
+  "env": {
+    "jest/globals": true
+  },
+  "extends": [
+    "plugin:react/recommended",
+    "airbnb",
+    "eslint:recommended",
+    "plugin:@typescript-eslint/recommended",
+    "plugin:jsx-a11y/recommended",
+    "prettier",
+    "prettier/react",
+    "prettier/@typescript-eslint",
+    "prettier/standard"
+  ],
+  "plugins": [
+    "@typescript-eslint",
+    "import",
+    "jest",
+    "jsx-a11y",
+    "prettier",
+    "react",
+    "tree-shaking"
+  ],
   "rules": {
-    "jsx-a11y/label-has-for": 0,
+    "@typescript-eslint/interface-name-prefix": ["warn", "never"],
+    "@typescript-eslint/explicit-function-return-type": "off",
+    "@typescript-eslint/no-empty-interface": [
+      "warn",
+      {
+        "allowSingleExtends": true
+      }
+    ],
+    "@typescript-eslint/no-explicit-any": "off",
+    "@typescript-eslint/no-inferrable-types": [
+      "warn",
```

Comment:
Not sure if it's a good idea to use warnings since those might not work in Circle CI. At least worth checking before merging this setup.

Reply:
So, should we use errors instead of warnings?
In this particular rule or in all rules? 

Ideally there is no warnings at all, so basically we could use just errors 🤔

Reply:
Does the CI pipeline pass with these settings?

Reply:
Seems to work so this is ok for now in my opinion!